src/cgame/default/cg_main.c

```diff
@@ -106,6 +107,8 @@ static void Cg_Init(void) {
 
 	cg_fov = cgi.Cvar("cg_fov", "110.0", CVAR_ARCHIVE, NULL);
 	cg_fov_zoom = cgi.Cvar("cg_fov_zoom", "55.0", CVAR_ARCHIVE, NULL);
+	cg_fov_interpolate = cgi.Cvar("cg_fov_interpolate", "100", CVAR_ARCHIVE,
+			"Speed to change the fov at in x100 degrees/second. 0 disables, 1 is 100 deg/sec, 0.5 is 50 deg/sec");
```

Comment:
@WickedShell I originally did that, but it can't work because it's stateless.
The zoom is just an alias to change the FOV, so it can go from anywhere to anywhere, including changing the FOV target while already in the interpolation state.
Having a fixed time to interpolate between the default FOV and zoom FOV would require the zooming to be remade as an FOV modifier/factor.